mitodl/mitxonline
# Made entire course card clickable

#### Pre-Flight checklist

 
#### What are the relevant tickets?
Ticket #103 

#### What's this PR do?
Makes entire course card clickable.

#### How should this be manually tested?
- Add course to home page through cms
- Verify that it redirects to detail page on click
